install/linux.md

```diff
@@ -81,7 +81,7 @@ Hier müssen im Terminal die folgenden Zeilen eingegeben werden:
 ```bash
 curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
 bash Mambaforge-$(uname)-$(uname -m).sh -p ~/.local/mambaforge -b
-source "${HOME}/.local/mambaforge/etc/profile.d/mamba.sh"
+source "${HOME}/.local/mambaforge/etc/profile.d/conda.sh"
```

Comment:
If you source `conda.sh`, `mamba activate` won't work.

Reply:
It should be the `mamba.sh` if you use `mamba activate` / `mamba init`

Reply:
True, I thought the same, but I had the same issue like Christian.

It did not find `conda` when just sourcing the `mamba.sh`.
And when I installed a clean docker ubuntu, it worked by just sourcing `conda.sh`.

Reply:
![image](https://github.com/pep-dortmund/homepage-toolbox/assets/41512963/8a229972-9d1c-4953-a5ed-aac2303ac001)


We have to do both.

Reply:
You did not comment but started a review. Then you have to finish before your comments are shown to us

Reply:
> What is the benefit of moving again, if we still end up with an active conda environment?

This is not the same as an anaconda installation. It is much lighter and mamba is -just a feeling- a thousand times faster than conda. In the end mamba is conda only way more efficient. And without being so heavy per default.

Reply:
> > It did not find conda when just sourcing the mamba.sh.
> 
> 
> 
> Yes. But for what do you need conda?

What I sent is from the original mamba website. I don't know why exactly they need it, but apparently they do.

There is an error if you don't source conda.sh and try to run mamba activate and mamba init

Reply:
`mamba init` also adds the sourcing of `conda.sh` to the shell init file. So the correct way I'd say is:

```
$ bash Mambaforge-$(uname)-$(uname -m).sh -p ~/.local/conda -b
$ source ~/.local/conda/etc/profile.d/mamba.sh
$ mamba init
```
and then close the current and open a new terminal, which will have everything setup correctly so that both `conda` and `mamba` work.